src/layouts/hass-tabs-subpage.ts

```diff
@@ -88,23 +87,23 @@ class HassTabsSubpage extends LitElement {
       return shownTabs.map(
         (page) =>
           html`
-            <ha-tab
-              .hass=${this.hass}
-              @click=${this._tabTapped}
-              .path=${page.path}
-              .active=${page === activeTab}
-              .narrow=${this.narrow}
-              .name=${page.translationKey
-                ? localizeFunc(page.translationKey)
-                : page.name}
-            >
-              ${page.iconPath
-                ? html`<ha-svg-icon
-                    slot="icon"
-                    .path=${page.iconPath}
-                  ></ha-svg-icon>`
-                : html`<ha-icon slot="icon" .icon=${page.icon}></ha-icon>`}
-            </ha-tab>
+            <a href=${page.path}>
```

Comment:
```suggestion
            <a href=${page.path} target=${this.supervisor ? "_parent" : "_self"}>
```

Reply:
Or we might just want to set the name for the parent, and use that as target:

```js
window.name = "ha-main";

...

<a target="ha-main">
```

Reply:
I don't understand, why do we need this ?